lib/elixir/lib/port.ex

```diff
@@ -3,56 +3,79 @@ defmodule Port do
   Functions related to Erlang ports.
   """
 
+  @type name :: {:spawn, string | binary} |
+                {:spawn_driver, string | binary} |
+                {:spawn_executable, string | atom} |
+                {:fd, non_neg_integer, non_neg_integer}
+
   @doc """
-  See [`:erlang.open_port/2`](http://www.erlang.org/doc/man/erlang.html#open_port-2)
+  Opens an Erlang port given a tuple `name` and a list of settings.
 
   Inlined by the compiler.
+
```

Comment:
Hi @eksperimental, thanks for the suggested improvement. I followed your directions and detailed a little more the different values of the parameter name for Port.open/2. I tried not to be too verbose in the explanation of each options, and didn't mention the PATH searching difference between {spawn, command} and {spawn_executable, filename}, as the need to use os:find_executable/1 to find the program in the path in the case of {spawn_executable, filename}.

I didn't know if we should mention it, or let the reader go to the Erlang doc to have more info. Thanks again.